### DISCUSSION TITLE
[Documentation] Update Discord FAQ

### DISCUSSION THREAD

**Discussion by gabrc52** [42wim/matterbridge#1322] [General]
I'm filing under bug, since I'm limited to only filing bugs and feature requests.

I'd add a new question:

**Q: Messages being sent to the wrong channel**

A: You're using webhooks; and the webhook editing functionality is currently broken (#1296). You should move the webhook to the right channel, and make a webhook per channel.

And remove the following question:

**Q: Messages not being edited, but a new message with (edit) appears?**

Message editing is not supported by discord using webhooks. If you want message editing, disable webhooks and use token only

Since message editing with webhooks *is* supported and works fine, this should be removed.

I can submit these changes as pull request, if they're okay as in this issue or there's a suggestion to rephrase it.

**Comment by gabrc52** [comment]
https://github.com/42wim/matterbridge/wiki/Section-Discord-%28basic%29 should be updated as well since webhook editing doesn't work even if the bot has the required permissions.

**Comment by qaisjp** [comment]
How do your plans change with the introduction of https://github.com/42wim/matterbridge/pull/1323?
